geth-dev-node/start.sh

```diff
@@ -0,0 +1,3 @@
+geth --password /app/password.txt account new
+geth init /app/genesis.json
+geth --password /app/password.txt --unlock 0 --mine --networkid=10000 --ws --wsapi eth,net,web3,personal --wsport 8546 --rpc --rpcapi eth,net,web3,personal,miner --rpcaddr 0.0.0.0 --targetgaslimit 6700000 --ethash.dagdir "/app/.ethash"
```

Comment:
What does the `--unlock 0` represent here?  I thought this was supposed to take in an address like `--unlock 0x00a329c0648769a73afac7f9381e08fb43dbea72` or something?

Reply:
It represents the index in the array of accounts.  (I believe you might also be able to use the word `primary`.)  It was simpler to use the index than trying to get the exact address generated from `geth --password /app/password.txt account new`.